mantis-control-plane/mantis-control-plane-client/build.gradle

```diff
@@ -17,10 +17,13 @@
 dependencies {
     implementation libraries.asyncHttpClient
     implementation libraries.spotifyFutures
+    implementation libraries.hadoopCommon
     api project(":mantis-control-plane:mantis-control-plane-core")
 
     api project(":mantis-remote-observable")
 
     testImplementation libraries.junit4
     testImplementation libraries.mockitoAll
+    testImplementation libraries.spectatorApi
+    testImplementation(testFixtures(project(":mantis-common")))
```

Comment:
Instead of doing this for every project, can we just put `testImplementation libraries.spectatorApi` in the root `build.gradle` for all subprojects? This way, we can remove it once rather than in all `build.gradle` files.